### DISCUSSION TITLE
Creating a project in cloud dashboard

### DISCUSSION THREAD

**Discussion by kazcw** [enso-org/enso#6928] [Help and troubleshooting] [answered]
When I click on a project template, nothing is visibly happening. This is because I have a long list of existing projects, and newly-created projects are inserted after all the older projects that have a "Last modified" date. It would be more intuitive if clicking on a project template added the project at the top of the list where I can see it appear.

**Comment by farmaazon** [comment, marked as answer]
I agree, had exactly the same effect: I've created 10 projects before realizing they are on the bottom of the list.

Let's make this an issue: [#6929](https://github.com/enso-org/enso/issues/6929)
